src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -62,35 +69,74 @@ function ProductDetails({ product: staleProduct }: Props) {
   })
 
   return (
-    <div>
+    <div className="grid-content product-details">
       <Breadcrumb breadcrumbList={breadcrumbs.itemListElement} />
-      <h2>{variantName}</h2>
-      <Image
-        src={productImages[0].url}
-        variant="product.details"
-        alt={productImages[0].alternateName}
-        loading="eager"
-      />
-      <div className="line-through">{formattedListPrice}</div>
-      <div className="min-h-[2rem]">{isValidating ? '' : formattedPrice}</div>
-      <SkuSelector
-        label="Size"
-        variant="label"
-        options={[
-          { label: 'P', value: 'p' },
-          { label: 'M', value: 'm' },
-          { label: 'G', value: 'g' },
-        ]}
-      />
-      {/* NOTE: A loading skeleton had to be used to avoid a Lighthouse's
-      non-composited animation violation due to the button transitioning its
-      background color when changing from its initial disabled to active state.
-      See full explanation on commit https://git.io/JyXV5. */}
-      {isValidating ? (
-        <AddToCartLoadingSkeleton />
-      ) : (
-        <Button {...buyProps}>Add to cart</Button>
-      )}
+
+      <section className="product-details__body">
+        <section className="product-details__title">
+          <ProductTitle
+            title={<h2 className="title-product">{name}</h2>}
+            label={<DiscountBadge listPrice={100} spotPrice={50} />}
+            refNumber="Ref.: 54412"
+          />
+        </section>
+
+        <section className="product-details__image">
+          <Image
+            src={productImages[0].url}
+            variant="product.details"
+            alt={productImages[0].alternateName}
+            loading="eager"
+          />
+        </section>
+
+        <section className="product-details__settings">
+          <section className="product-details__values">
+            <div className="prices">
```

Comment:
I think you can use our `Price` component here! See an example on `ProductCard` component.